File: Iniciante/URI 1044.hs
main :: IO ()
main = do
    line <- getLine
    let (a:b:_) = map (read :: String -> Int) $ words line

    if a `mod` b == 0 || b `mod` a == 0
        then putStrLn $ "Sao Multiplos"
    else
        putStrLn $ "Nao sao Multiplos"

File: Iniciante/URI 1044.lua
line = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line[n] = i
end
a = tonumber(line[1])
b = tonumber(line[2])

if (a % b == 0 or b % a == 0) then
    print('Sao Multiplos')
else
    print('Nao sao Multiplos')
end